cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/CaReconciler.java

```diff
@@ -522,7 +536,7 @@ Future<Void> rollingUpdateForNewCaKey() {
                 });
     }
 
-    /* test */ Future<Void> rollKafkaBrokers(Set<NodeRef> nodes, RestartReasons podRollReasons) {
+    /* test */ Future<Void> rollKafkaBrokers(Set<NodeRef> nodes, RestartReasons podRollReasons, TlsPemIdentity kafkaTlsPemIdentity) {
```

Comment:
Same as above -> and possibly in other places as well. This is not a Kafka identity -> it is a Cluster operator identity, or?